### PR TITLE
Add resize_coordinate_window tool

### PR DESCRIPTION
**Pull request by gregvonkuster**
This tool will perform steps 2,3 and 4 of this workflow https://usegalaxy.org/u/memesuite/w/create-meme-chip-input-fasta-file-500bp-centered-regions-from-macs-summitsbed-file in a single step for gff data.

### CODE REVIEW

**Comment by gregvonkuster** [comment]
ping @blankenberg @martenson @bgruening Dan, you and I have discussed this tool.

**Comment by martenson** [comment]
lgtm

File: tools/resize_coordinate_window/resize_coordinate_window.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+parser = argparse.ArgumentParser()
+parser.add_argument('--input', dest='input', help="Input dataset")
+parser.add_argument('--subtract_from_start', dest='subtract_from_start', type=int, help='Distance to subtract from start.')
+parser.add_argument('--add_to_end', dest='add_to_end', type=int, help='Distance to add to end.')
+parser.add_argument('--extend_existing', dest='extend_existing', help='Extend existing start/end rather or from computed midpoint.')
+parser.add_argument('--output', dest='output', help="Output dataset")
+args = parser.parse_args()
+
+extend_existing = args.extend_existing == 'existing'
+out = open(args.output, 'wb')
```
Comment (bgruening) [inline tools/resize_coordinate_window/resize_coordinate_window.py]: You could use `with` here.

**Comment by bgruening** [comment]
@blankenberg please feel free to join.

**Comment by gregvonkuster** [comment]
Ping @martenson @blankenberg @bgruening Is there anything remaining that I should do in order to get this PR accepted and proceed with getting this tool available on the Main Tool Shed?  Thanks!

**Comment by bgruening** [comment]
I'm fine with this. Wanted to wait for @blankenberg, but if he is busy I will merge it in the evening.

**Comment by gregvonkuster** [comment]
@bgruening Thanks, I need this for the ChIP-exo flavor!  ;)

**Comment by gregvonkuster** [comment]
@markiskander @bgruening I'm fairly certain thet @blankenberg is not going to have any input on this.  Can it be merged and uploaded to the MTS please?

**Comment by gregvonkuster** [comment]
Awesome!  Thanks @bgruening !!  ;)

**Pull request merged**